frontend/dockerfile/dockerfile_test.go

```diff
@@ -1593,14 +1610,21 @@ FROM busybox
 }
 
 func testArgDefaultExpansion(t *testing.T, sb integration.Sandbox) {
-	integration.SkipOnPlatform(t, "windows")
 	f := getFrontend(t, sb)
 
-	dockerfile := []byte(`
+	dockerfile := []byte(integration.UnixOrWindows(
+		`
 FROM scratch
 ARG FOO
 ARG BAR=${FOO:?"foo missing"}
-`)
+`,
+		`
+FROM nanoserver AS build
+ARG FOO
+ARG BAR
```

Comment:
Same there, why not `ARG BAR=${FOO:?"foo missing"}`?

Reply:
I've reverted this change so that we can investigate this further - same as the previous comment.